src/HDInsight/HDInsightOnAks.Autorest/custom/New-AzHdInsightOnAksClusterConfigFile.ps1

```diff
@@ -0,0 +1,60 @@
+# ----------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# ----------------------------------------------------------------------------------
+
+
+function New-AzHdInsightOnAksClusterConfigFile {
+[OutputType([Microsoft.Azure.PowerShell.Cmdlets.HdInsightOnAks.Models.Api20230601Preview.IClusterConfigFile])]
+[CmdletBinding(DefaultParameterSetName='Create', PositionalBinding=$false)]
+param(
+    [Parameter(ParameterSetName='Create', Mandatory)]
+    [Alias('Name')]
+    [System.String]
+    # The name of the config file.
+    ${FileName},
+
+    [Parameter(ParameterSetName='Create', Mandatory)]
+    [System.Collections.HashTable]
```

Comment:
Please add block comments for function as below and add parameter comments for Value otherwise the description from them are empty. Please check other customized cmdlets as well.

![image](https://github.com/Azure/azure-powershell/assets/19869716/f9b29543-4145-49e0-b070-9dead9cda741)


![image](https://github.com/Azure/azure-powershell/assets/19869716/c2dc7971-0012-425a-8430-2a8f1ce62df5)

Reply:
> Please add block comments for function as below and add parameter comments for Value otherwise the description from them are empty. Please check other customized cmdlets as well.
> 
> ![image](https://user-images.githubusercontent.com/19869716/267925136-f9b29543-4145-49e0-b070-9dead9cda741.png)
> 
> ![image](https://user-images.githubusercontent.com/19869716/267924807-c2dc7971-0012-425a-8430-2a8f1ce62df5.png)

Fixed